src/components/Book.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Book = () => (
+  <div className="App">
+    <h2>Book Lists</h2>
+    <form>
+      <div>
+        <input type="text" />
+      </div>
+      <button type="submit">Add Book</button>
+    </form>
+  </div>
+);
```

Comment:
- [ ] It will be good to have another component for a single book and another component for the form, that way you'll render those two components in this BookList Component.i.e. you'll have something like this
```js
const Book = () => (
  <div className="App">
    <Book book={book}/>
    <Form />
 </div>
);
```
- [ ] The Book component should take the book data as a prop and should render the title, the author and a button for removing books. The button wouldn't remove the books yet, this is just a placeholder for now.
- [ ] The Form component should render a form with inputs for the title, the author of the book and a button to submit. The button wouldn't add the book *yet*, this is just a placeholder for now.